examples/visual-tests/amp-story/amp-story-shopping.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html amp lang="en">
+<html amp lang="en" dir="rtl">
```

Comment:
Please see previous comment about adding rtl to the amp-story-page element that has the rtl examples.